### ISSUE TITLE
Vim Chord and Vim Mode display freeze / stop working when switched to a new note, on another tab and after some time

### ISSUE THREAD

**Issue by DonDon-9097** [esm7/obsidian-vimrc-support#149]
### **Bug:**
Vim Chord and Vim Mode display freezes and stops reacting to chords and mode state, when switched to a new note, on another tab and after some time

![image](https://user-images.githubusercontent.com/29581769/201424761-604750d4-a6ae-498d-8ee6-aba390454936.png)

Since it ceases to work when the window changes I think there is something not working with "initializing" the display on the note workspace?

The only method I found to make it work again is to reset obsidian

### **To Reproduce:**

- leave Obsidian in reading mode for some time (I tested around 15min)
- change a note in a tab (chords and display will stop working even when coming back to last note)
- change the tab (chords and display won't work on the new tab, but will still work in the oryginal tab)

### **System info**
 - OS: Windows 10
 - Vimrc plugin version: 0.8.0

### **Additional context:**

I tested this on my main vault and on a fresh vault with stock settings and no other plugins.

**Comment by Shenliuran** [comment]
@DonDon-9097 I have the same problem, did you solve it?

**Comment by DonDon-9097** [comment]
I missclicked, the bug is still up.
 I'm not really a dev, so I haven't looked into this bug, it does annoy me, but I'd have to research a lot to understand why it doesn't work... since it stops working on a second tab, or after a longer incactivity I'd guess it's not liked as a process that shoud start on the tab start/re-focus.
I've spent a little bit of time looking around and I think it's a bug inside the code, not something in options/system.
